x-pack/plugins/monitoring/server/config.ts

```diff
@@ -58,7 +58,6 @@ export const configSchema = schema.object({
     }),
   }),
   cluster_alerts: schema.object({
-    allowedSpaces: schema.arrayOf(schema.string(), { defaultValue: ['default'] }),
```

Comment:
@smith mentioned we might want to keep this as an unused/deprecated value https://github.com/elastic/kibana/blob/6693ef371f887eca639b09c4c9b15701b4ebabd4/docs/developer/architecture/core/configuration-service.asciidoc#handle-plugin-configuration-deprecations

Though I think @jasonrhodes mentioned it wasn't actively used to begin with. Maybe fine to remove it directly.

Reply:
Flagged the setting as `unused` which sounds like a better ux